src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  <p>
```

Comment:
According to check list we have to write arguments in one line if there are less than three. But anyway good job!!!